### DISCUSSION TITLE
"Jugememt" giltch?

### DISCUSSION THREAD

**Discussion by murryguy** [ppy/osu#28156] [Q&A]
### Type

Game behaviour

### Bug description

Once I click this, I miss. Is this a glitch??

### Screenshots or videos

![Screenshot 2024-05-11 102317](https://github.com/ppy/osu/assets/167148240/01bf9942-2e7f-4168-8ea4-c8c9fa27bc4e)

### Version

Lazer 2024.142.1

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/15283689/compressed-logs.zip)

**Comment by Gabixel** [comment]
I'm sorry, but a screenshot of this size doesn't tell us much about what's happening. Are you sure you are not hitting the circles too early/late?

Unfortunately, the logs don't seem to provide any information about this "glitch".
A good start would be to include the beatmap where you're experiencing the issue, or a video... any additional details.

(also, for future reference: there's no need to post in both issues [and discussions](https://github.com/ppy/osu/discussions/28151). To put it simply, you can "choose" between the two. See the [guidelines](https://github.com/ppy/osu/blob/master/CONTRIBUTING.md#issue-or-discussion) for more info)

**Comment by murryguy** [comment]
> I'm sorry, but a screenshot of this size doesn't tell us much about what's happening. Are you sure you are not hitting the circles too early/late?
> 
> Unfortunately, the logs don't seem to provide any information about this "glitch". A good start would be to include the beatmap where you're experiencing the issue, or a video... any additional details.
> 
> (also, for future reference: there's no need to post in both issues [and discussions](https://github.com/ppy/osu/discussions/28151). To put it simply, you can "choose" between the two. See the [guidelines](https://github.com/ppy/osu/blob/master/CONTRIBUTING.md#issue-or-discussion) for more info)

I had to make a screenshot becuase the video was over 10MB.

**Reply by Gabixel** [reply]
In that case see if you can reduce the quality even more to make it fit here, otherwise you can try some external service (like [Streamable](https://streamable.com/), or even Google Drive or Dropbox, or YouTube, just to name a few)
